### DISCUSSION TITLE
?will opentrack ever track multiple objects

### DISCUSSION THREAD

**Discussion by rackhamlerouge9** [opentrack/opentrack#1767] [General]
Hi all,  very deepest thanks to the devs for their incredible work with opentrack all the way from concept to celebrated an maintained program.

I'm curious to know if opentrack's scope will always be *just* head tracking, or if it will ever expand to track the movement of anything else our imagination allows us to stick sensors or markers to (hands/feet/cats/aeroplanes).

Is this something about which the devs have strong views?

Am I behind the curve, and there are already ways to use opentrack to track more than one object? (If so, apologies, I did my best web-searching before I came to ask, I promise!)

If not, and/or there's any other FOS Software that already tracks multiple objects (especially aruco markers or fiducial objects!) I'd be glad of the steer.

Thanks for your consideration,
RLR

**Comment by sthalik** [comment]
How would opentrack's code even help you with it? You can use our forked version of Aruco and the video input modules without bringing in the rest of opentrack.

**Reply by rackhamlerouge9** [reply]
Top thinking - thank you.  And thank you again for all your work keeping all this spinning!
